.github/workflows/release.yml

```
@@ -33,5 +35,49 @@ jobs:
           name: nugets
           path: nugets/*
           retention-days: 1
-      - name: Deploy to MyGet
-        run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}
+     #- name: Deploy to MyGet
+       #run: dotnet nuget push nugets/Particular.EndpointThroughputCounter.${{env.MinVerVersion}}.nupkg --source https://www.myget.org/F/particular/api/v2/package --api-key ${{ secrets.MYGET_PUBLISH_API_KEY }}       
+      - name: Build SelfContained
+        run: >
+          dotnet publish src/Tool/Particular.EndpointThroughputCounter.csproj -r win-x64
+          -c Release
+          -p:TargetFramework=net6.0
+          -p:PublishSingleFile=true
+          -p:SelfContained=true
+          -p:PublishReadyToRun=true
+          -p:IncludeAllContentForSelfExtract=true
+          -p:TargetLatestRuntimePath=true
+          -p:DebugType=embedded
+          -p:EnableCompressSingleFile=true
+          -p:PackAsTool=false
+          -o ./src/binaries    
+      - name: Sign Exe
+        shell: pwsh
+        run: >
+              NuGetKeyVaultSignTool sign  src/binaries/Particular.EndpointThroughputCounter.exe
+              --file-digest sha256
+              --timestamp-rfc3161 http://timestamp.digicert.com
+              --timestamp-digest sha256
+              --azure-key-vault-url https://particularcodesigning.vault.azure.net
+              --azure-key-vault-client-id ${{ inputs.client-id }}
+              --azure-key-vault-tenant-id ${{ inputs.tenant-id }}
+              --azure-key-vault-client-secret ${{ inputs.client-secret }}1
+              --azure-key-vault-certificate ${{ inputs.certificate-name }}
```

`inputs` are because this was taken from our custom action. The values come from where we use it, like https://github.com/Particular/NServiceBus/blob/master/.github/workflows/release.yml#L26-L29 so I think this will work if we use the already org-level secrets directly.

And I think that `1` is not supposed to be on the end there.
```suggestion
              --azure-key-vault-client-id ${{ secrets.AZURE_KEY_VAULT_CLIENT_ID }}
              --azure-key-vault-tenant-id ${{ secrets.AZURE_KEY_VAULT_TENANT_ID }}
              --azure-key-vault-client-secret ${{ secrets.AZURE_KEY_VAULT_CLIENT_SECRET }}
              --azure-key-vault-certificate ${{ secrets.AZURE_KEY_VAULT_CERTIFICATE_NAME }}
```